src/main/java/org/privacyidea/Endpoint.java

```diff
@@ -60,7 +61,7 @@
 class Endpoint {
 
     private final PrivacyIDEA privacyIDEA;
-    private List<String> logExcludedEndpointPrints = Collections.emptyList(); //Arrays.asList(PIConstants.ENDPOINT_AUTH, PIConstants.ENDPOINT_POLLTRANSACTION); //
+    private List<String> logExcludedEndpointPrints = Arrays.asList(PIConstants.ENDPOINT_AUTH, PIConstants.ENDPOINT_POLLTRANSACTION); //Collections.emptyList(); //
```

Comment:
why do You exclude these from logging? Because the resulting JSON contains sensible data? Actually this data could be useful for debugging (But we currently do not have a possibility to categorize log messages?

Reply:
because the poll endpoint spams the log and the the auth endpoint response is really long and does not really matter, if there is no auth token or response is empty an error is logged anyway. This is just kind of a development setting. When using the sdk, the app can set this list to whatever it wants. I will overwrite this in keycloak just to make sure. Currently the log messages are error or log, but the sdk only uses an interface, the app has to the map the interface to the logger that the app has access to.